moco.py

```diff
@@ -112,7 +56,10 @@ def __init__(
         elif isinstance(hparams, dict):
             hparams = self.params(**hparams, **kwargs)
 
-        self.hparams = AttributeDict(attr.asdict(hparams))
+        if pl.__version__ >= '1.3.0':
```

Comment:
I'm not sure this comparison is the right way to check the version, perhaps use 

`if isinstance(self.hparams, AttributeDict):`

Reply:
Good point, string comparison is too brittle. I added your fix, which works for all versions of pytorch-lightning I tested.